### ISSUE TITLE
bed files for databases

### ISSUE THREAD

**Issue by jameskozubek** [richardslab/Ei#3]
Hello,

I'm having trouble with the 02_annotate.sh script, as it seems the database bed files (dbsnp, genes, chromatin, gtex, etc) are custom built and it's hard to know what fields are available, and how many fields are used in formulation of those bed files. Thereafter, they are referred to by awk commands that identify some of the fields by number. 

Also I don't know what $TRAIT means and how to set that in this script, or what trait.dir <- "M01494" means in the next 03 script. 

In short, I created a number of bed files by downloading the dbsnp, gene tracks, and etc. but I run into problems down the road no knowing exactly what the files should have in them. Is it possible to make available some toy data and the bed files for the databases for download? If not, no worries, and thank you for the code.

**Comment by richardslab** [comment]
I will tackle this over a few messages as I solve each issue for you. 

* For the DHS data, this is largely  available from http://www.mauranolab.org/CATO/, under "Master list of DHSs".
* For genes, I parsed GENCODE for protein coding genes and formatted results to BED format. I will post code and file for this shortly.
* For GTEX, I downloaded v7 and did some post-processing which I will explain also shortly.
* Same for dbSNP, for which I downloaded the table from UCSC.

For the $TRAIT variable, this is a small error. The code should read that parameter from the command line.

Of note, I am in the midst of significant refactoring of code to remove all these dependencies, ie to have the entire pipeline in a single bundle. I hope to have this completed in a few weeks.

**Comment by jameskozubek** [comment]
okay, maybe it makes sense to wait for a bundle. If you are going to create a pipeline in a single bundle, we can wait rather than trouble you for the individual processing steps.

**Comment by gustavahlberg** [comment]
How did you post process: 
GTEX v7, dbSNP, ENCODE_REMC_FLER_T2D_2019aug.flt.hotspots2.fdr0.05.hg19.bed... etc?

**Comment by jameskozubek** [comment]
Thank you, sir. However, I do not recall at this time.

**Comment by gustavahlberg** [comment]
Would it be possible to share a resource bundle if available?
